src/components/Icon/__tests__/Icon.spec.ts

```diff
@@ -4,12 +4,14 @@ import Icon from '../Icon.vue';
 import { IconName } from '../types';
 
 describe('Icon', () => {
-  it.each([...Object.values(IconName)])('renders', async (iconName) => {
-    const { findByTestId } = render(Icon, {
-      props: {
-        icon: iconName
-      }
+  [...Object.values(IconName)].forEach((iconName) => {
+    it(`renders ${iconName} icon`, async () => {
```

Comment:
It's nice to have the icon name in the test output 😃

Reply:
This makes things too easy for us though..